site/topics/introduction/introduction.rst

```diff
@@ -2,12 +2,112 @@
+
+.. figure:: transistors.png
+    :width: 333 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Transistor
+
+    A collection of four transistors. Although these transistors are all physically different, their function is the
+    same.
+
+
+* There are several layers of abstraction between transistors and programming that will be covered
+
+    * Transistors
+    * Logic Gates
+    * Latches
+    * Registers
+    * Memory
+    * Adders
+    * Arithmatic Logic Units (ALUs)
+    * Combinatorial and Sequential Logic
+    * Control Logic
+    * CPUs
+    * Architectures
+    * Machine Language/Assembly Language
+    * Compilers
+
+.. figure:: von_neumann_architecture.png
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Von_Neumann_architecture
+
+    High-level idea of the von Neumann architecture. One of the interesting features of this architecture is how memory
+    stores both data and instructions.
+
+
+.. figure:: harvard_architecture.png
+    :width: 500 px
+    :align: center
+    :target: https://en.wikipedia.org/wiki/Von_Neumann_architecture
+
+    High-level idea of the Harvard architecture. This architecture has separate memory for data and instructions.
+
+
+
+
+Additional Resources
+--------------------
+
+* The course content is available on `GitHub <https://github.com/jameshughes89/cs2XX-ComputerArchitecture>`_
+* This course makes use of *Digital*
+
+    * Free software for simulating digital circuits
+    * Available on `GitHub <https://github.com/hneemann/Digital>`_
+
+.. figure:: digital_screenshot.png
+    :width: 500 px
+    :align: center
+
+    Screenshot of a small portion of a modified version of the 8-bit computer designed by Ben Eater. The larger circuit
+    is a simulated fully functional and programmable turing complete computer.
+
+
+
+* `Ben Eater has an amazing YouTube playlist where they build a turing complete computer on breadboards <https://www.youtube.com/playlist?list=PLowKtXNTBypGqImE405J2565dvjafglHU>`_
+
+    * These videos are fantastic and provide more details on components than will be covered in this course
+
+
+.. figure:: breadboard_computer.png
+    :width: 500 px
+    :align: center
+
+    Part of the 8-bit breadboard computer, as designed by Ben Eater.
+
+
+* The game `Turing Complete <https://store.steampowered.com/app/1444480/Turing_Complete/>`_
+
+    * Players incrementally construct and program a turing complete computer
+
+
+.. figure:: turing_complete_screenshot.png
+    :width: 500 px
+    :align: center
+
+    Screenshot of an ALU from a larger computer made within the game "Turing Complete".
+
 
 
 For Next Time
 =============
 
 * Get your computer at home :doc:`set up for the course </getting-set/getting-set>`
+* `Watch Ben Eater's 8-bit computer overview video <https://www.youtube.com/watch?v=HyznrdDSSGM&list=PLowKtXNTBypGqImE405J2565dvjafglHU&index=1>`_
+* Play the level "Crude Awakening" from the *Turing Complete* game
```

Comment:
_play a game_ -- think it's a bit unreasonable to ask students to buy a game. Mentioning the game is one thing, but giving homework to play it feels off. Potentially legally questionable too.